### ISSUE TITLE
Spastic monoplegia

### ISSUE THREAD

**Issue by peupeubangbang** [obophenotype/human-phenotype-ontology#7653]
**Preferred term label:**
Spastic monoplegia

**Synonyms**

**Definition (free text, please give PubMed ID)**
Spastic weakness affecting only one limb.
PMID: 24111874

**Parent term (use hpo.jax.org/app)**
Appendicular Spasticity HP:0034353
~~Cerebral palsy HP:0100021~~

**Xrefs**
SNOMEDCT_US: 56409008
UMLS: C0154698

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Comment by pnrobinson** [comment]
Is this typically meant to refer to a leg or should we keep this open?
Would Spastic lower-limb monoplegia be good?

**Comment by peupeubangbang** [comment]
@pnrobinson suggest keeping open as CP type classification considers limbs affected separately

this may end up getting _spastic cp unilateral_ as a parent but i will see what classification experts say next week

**Comment by pnrobinson** [comment]
I meant by "open" to not specify which limb was meant...but let's touch bases later about this

**Comment by peupeubangbang** [comment]
@pnrobinson confirming best to keep 'open' / limb agnostic

**Comment by peupeubangbang** [comment]
@pnrobinson are we able to still get this added as a CP motor phenotype? I have revised the parent term  **Appendicular spasticity** and now excluded CP as a parent

**Comment by pnrobinson** [comment]
@peupeubangbang sorry for the delay - I was relocating to Germany in 2023 and we got way behind.

Adding new term: 
- ID: HP:6000316
- label: Spastic monoplegia
- parents: [Spasticity (HP:0001257)](https://hpo.jax.org/app/browse/term/HP:0001257)
- pmid: PMID:24111874
- def: Spastic weakness found in only one upper or lower extremity.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7653
- synonyms:
